packages/block-editor/src/components/block-toolbar/use-has-block-toolbar.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { getBlockType, hasBlockSupport } from '@wordpress/blocks';
+/**
+ * Internal dependencies
+ */
+import { store as blockEditorStore } from '../../store';
+import { useHasAnyBlockControls } from '../block-controls/use-has-block-controls';
+
+/**
+ * Returns true if the block toolbar will be shown.
+ *
+ * @return {boolean} Whether the block toolbar component will be rendered.
+ */
+export function useHasBlockToolbar() {
```

Comment:
```suggestion
export function useShowBlockToolbar() {
```

Reply:
This isn't a check to know if it will show or not, but rather if it can be shown. Like, does the current selection _have a block toolbar it can render_? `useShowBlockToolbar` makes me think it will do something to display the toolbar.